fe1-web/src/features/digital-cash/screens/SendReceive/SendReceive.tsx

```diff
@@ -256,6 +256,10 @@ export const SendReceiveHeaderRight = () => {
 
   const publicKey = useMemo(() => rollCallToken?.token.publicKey.valueOf() || '', [rollCallToken]);
 
+  const copyPkToClipBoard = async () => {
+    await Clipboard.setStringAsync(publicKey);
+  };
```

Comment:
seeing here (and elsewhere) that we're creating an async function (hence returning a promise) to await a call that returns a promise.... can't we just call `Clipboard.setStringAsync()` in the `onPress`. removing entirely this function ? It might be simpler 😄